proguard/proguard-square-okhttp.pro

```diff
@@ -0,0 +1,2 @@
+# OkHttp3
+-dontwarn okhttp3.**
```

Comment:
we'll need more rules. there's one more for the publicsuffixdatabase thing. they've got it documented.